.sauce/playwright.yml

```diff
@@ -1,18 +1,30 @@
 apiVersion: v1alpha
-metadata:
-  name: Testing Playwright Support
-  tags:
-    - e2e
-  build: "$BUILD_ID"
-files:
-  - ./tests/e2e/playwright/example.test.js
+kind: playwright
+sauce:
+  region: us-west-1
+  concurrency: 1
+  metadata:
+    name: Testing Playwright Support
+    tags:
+      - e2e
+    build: "$BUILD_ID"
+docker:
+  fileTransfer: mount
+  image:
+    name: saucelabs/stt-playwright-jest-node
+    tag: latest
+playwright:
+  projectPath: tests/e2e/playwright/
+  version: 1.7.1
+  params:
+    browserName: firefox
 suites:
   - name: "saucy test"
-    match: ".*.(spec|test).js$"
-    settings:
+    platformName: "Windows 10"
+    testMatch: '**/*.js'
```

Comment:
Hm, for jest `testMatch` used to be an array. Is that not true for folio?

Reply:
According to folio, it's only one script given as a parameter.